Content.Client/Entry/IgnoredComponents.cs

```diff
@@ -323,7 +323,8 @@ public static class IgnoredComponents
             "GrowingKudzu",
             "MonkeyAccent",
             "ReplacementAccent",
-            "ResistLocker"
+            "ResistLocker",
+            "Esword"
         };
```

Comment:
Just a tip for the future, as you already noticed with ﻿`ResistLocker`, people often add stuff to the bottom of the list which leads to conflicts. That's why most people just throw new entries somewhere in the middle (which is why this is a horrible non-alphabetic mess).

Reply:
GODLY TIP. THANK YOU KINDLY, SIR!